tests/derive.test.tsx

```diff
@@ -211,7 +211,7 @@ it('nested emulation with derive', async () => {
     {
       doubled: (get) => computeDouble(get(state.math).count),
     },
-    { proxy: state.math }
+    { proxy: state.math, sync: true }
```

Comment:
This is technically a behavioral change, but hope it's not too critical.

Reply:
So if it's async it can't be glitch free?

Reply:
No, but I didn't confirm it in the nested scenario. Without `sync` in this case, it simply double call the subscriber. It's a side effect of supporting glitch free.